src/components/search/search.jsx

```diff
@@ -0,0 +1,251 @@
+					IconClasses,
+					'bg-background-primary border border-solid border-border-subtle',
+					dimension === 'sm'
+						? 'px-2 py-0.5'
+						: dimension === 'md'
+							? 'px-3 py-1'
+							: 'px-3.5 py-1'
+				)}
+			>
+				/
+			</span>
+
+		</div >
+	);
+});
+SearchBoxInput.displayName = 'SearchBoxInput';
+
+const SearchBoxContent = forwardRef(({
+	className,
+	dropdownPortalRoot = null, // Root element where the dropdown will be rendered.
+	dropdownPortalId = '', // Id of the dropdown portal where the dropdown will be rendered.
+	children, ...props
+}, ref) => {
+	const { open, refs, floatingStyles, getFloatingProps } = useSearchContext();
+
+	if (!open) return null;
+
+	return (
+		<FloatingPortal id={dropdownPortalId} root={dropdownPortalRoot}>
+			<div
+				ref={refs.setFloating}
+				style={{
+					...floatingStyles
+				}}
+				className={cn(
+					'bg-background-primary rounded-md p-2 border-border-strong shadow-lg overflow-y-auto',
+					className
+				)}
+				{...getFloatingProps}
+				{...props}
+			>
+				{children}
+			</div>
+		</FloatingPortal>
+	);
+});
+
+SearchBoxContent.displayName = 'SearchBoxContent';
+
+const SearchBoxLoading = ({ loadingIcon = <Loader /> }) => {
+	const { dimension } = useSearchContext();
+	return (
+		<div className={cn("justify-center p-4", textSizeClassNames[dimension])}>
+		{loadingIcon}
+	</div>
+	)
+};
+
+const SearchBoxResults = forwardRef(({ className, children, ...props }, ref) => (
+	<div ref={ref} className={cn('space-y-1', className)} {...props}>
+		{children}
+	</div>
+));
+SearchBoxResults.displayName = 'SearchBoxResults';
+
+const SearchBoxResultTitle = forwardRef(({ className, children, ...props }, ref) => {
+	const { dimension } = useSearchContext();
+
+	return (
+		<div
+			ref={ref}
+			className={cn(
+				'flex p-1',
+				className
+			)}
+			{...props}
+		>
+			<label className={cn("w-full text-text-secondary", textSizeClassNames[dimension])}>{children}</label>
+		</div>
+	)
+});
+SearchBoxResultTitle.displayName = 'SearchBoxResultTitle';
+
+const SearchBoxResultItem = forwardRef(({ className, children, icon, ...props }, ref) => {
+	const { dimension } = useSearchContext();
+	return (<div
+		ref={ref}
+		className={cn(
+			'flex items-center gap-2 p-1 hover:bg-gray-200',
+			textSizeClassNames[dimension],
+			!icon && "pl-4",
+			className
+		)}
+		{...props}
+	>
+		{icon}
+		<label className={cn(textSizeClassNames[dimension], "w-full text-text-secondary cursor-pointer")}>{children}</label>
+	</div>)
+});
+SearchBoxResultItem.displayName = 'SearchBoxResultItem';
+
+const SearchBoxSeparator = forwardRef(({ className, ...props }, ref) => (
+	<hr ref={ref} className={cn('w-full text-text-tertiary', className)} {...props} />
+));
+SearchBoxSeparator.displayName = 'SearchBoxSeparator';
+
+export {
+	SearchBox,
+	SearchBoxInput,
+	SearchBoxContent,
+	SearchBoxLoading,
+	SearchBoxResults,
+	SearchBoxResultTitle,
+	SearchBoxResultItem,
+	SearchBoxSeparator,
+};
```

Comment:
Please follow how we are exporting other components.

**Export:**
```jsx
export default SearchBox
```

**Import and usage:**
```jsx
import {SearchBox} from '@bsf/force-ui'
<SearchBox>
...
<SearchBox.Input />
...
</SearchBox>
```

Reply:
Fixed!